src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,430 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""bravyi_kitaev_fast transform on fermionic operators."""
+from __future__ import absolute_import
+from fermilib.ops import InteractionOperator
+import networkx
+import numpy
+from projectq.ops import QubitOperator
```

Comment:
Imports are best structured by
1. Python modules
2. External modules
3. FermiLib stuff
(4. the implemenation file, if this would be the test for it)
with one space inbetween